### PR TITLE
lock the numpy version

### LINKED ISSUE #117: numpy dependency needs to be locked
numpy==1.15.3

### LINKED ISSUE #120: Deprecation warning
```
/[path snipped]/aggregation-for-caesar/panoptes_aggregation/reducers/text_utils.py:72: FutureWarning: arrays to stack must be passed as a "sequence" type such as list or tuple. Support for non-sequence iterables such as generators is deprecated as of NumPy 1.16 and will raise an error in the future.

  overlap_lines = np.vstack({tuple(row) for row in overlap_lines})
```

### PR DESCRIPTION
**Pull request by camallen**
closes #117, fixes #120  - fix a numpy dependency error

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -76,7 +76,7 @@
         'collatex==2.1.2',
         'hdbscan',
         'lxml',
-        'numpy>=1.14.5',
+        'numpy==1.15.0',
```
Comment (camallen) [inline setup.py]: I had to bump this to `1.16.0` to get it working with db scan - https://github.com/scikit-learn-contrib/hdbscan/issues/272

**Comment by CKrawczyk** [comment]
I am going to run some tests with locking to `1.16.2` (the current latest version) in a clean `env` to see if the bug we ran into during the workshop still happens.

File: setup.py (hunk 1 above)
Comment (CKrawczyk) [reply setup.py]: You can safely change this to `numpy==1.16.2`, I have tested it in a clean (conda) environment on my mac and run through the full workshop example without issue.  `skimage` and `hdbscan` both work as expected numpy's latest patch.

Comment (camallen) [reply setup.py]:
```suggestion
        'numpy==1.16.2,
```

**Comment by camallen** [comment]
@CKrawczyk updated numpy, fixed the deprecation warning and merged in the latest master code. This should be good to go pending specs.

**Pull request merged**